package.json

```
@@ -75,7 +75,7 @@
     "@testing-library/react-hooks": "^7.0.1",
     "@types/enzyme": "^3.10.8",
     "@types/express": "^4.17.13",
-    "@types/jest": "^26.0.23",
+    "@types/jest": "27.5.2",
```

```suggestion
    "@types/jest": "^27.5.2",
```

Same with `"@types/ua-parser-js"` while we're here.